### ISSUE TITLE
storing weight for each instance in mask rcnn

### ISSUE THREAD

**Issue by jamesbc123** [open-mmlab/mmdetection#8741]
Hello MMdetection Team,

I would like to keep track of a weight for each instance during training with mask rcnn. It could e.g. start at 1.0 for each instance, but then after each epoch it will change (by me changing it) and I want to store the updated weights somewhere. I have looked into Models/losses/crossentropyloss.py and found that there is a sample_weight. But where should I return the weight to (from the loss function) to keep it saved for the next epoch?

**Comment by jamesbc123** [comment]
And how can I get the instance id so I know which weight belongs to which instance?
